diff --git a/src/timing/formatter/mod.rs b/src/timing/formatter/mod.rs
index e400ef3..07c71f4 100644
--- a/src/timing/formatter/mod.rs
+++ b/src/timing/formatter/mod.rs
@@ -58,9 +58,9 @@ pub trait TimeFormatter<'a> {
 
 const EPSILON: f64 = 0.000_000_1;
 /// The dash symbol to use for generic dashes in text.
-pub const DASH: &str = "—";
+pub const DASH: &str = "-";
 /// The minus symbol to use for negative numbers.
-pub const MINUS: &str = "−";
+pub const MINUS: &str = "-";
 /// The minus symbol to use for negative numbers, where the minus symbol
 /// specifically needs to be the ASCII minus.
 pub const ASCII_MINUS: &str = "-";
